package.json

```diff
@@ -25,6 +25,7 @@
   "private": true,
   "dependencies": {
     "@hapi/boom": "9.1.0",
+    "@hcaptcha/react-hcaptcha": "0.2.2",
```

Comment:
Gdzie tego używamy?

Reply:
Mieliśmy to użyć w formularzu, ale dokumentacja nie była zbyt opisowa i narazie zdecydowałem zrobić bez captchy.